### DISCUSSION TITLE
Welcome to terraform-provider-snowflake Discussions!

### DISCUSSION THREAD

**Discussion by ryanking** [Snowflake-Labs/terraform-provider-snowflake#261] [General]
## 👋 Welcome everyone!

I would like to try using this new Discussions feature to connect with folks who use and work on this provider. I hope we can use this space, rather than issues to:

  * Ask for help
  * Share ideas for new features or implementation approaches
  * learn, develop and share best practices
  * get to know each other.

To get started, comment below with an introduction and tell us about what your involvement and interest is in this project.

**Comment by sjauld** [comment]
Nice one @ryanking! 

I'm Head of Infrastructure at an Australian video streaming company - Viostream. We store bandwidth and click stream data in Snowflake and are starting to give our customers access to their data via reader accounts. 

This provider helps me to automate the setup of those accounts, and my main activity here has been to add resources that were missing, although I don't have as much time as I would like to get it done 🤨

**Comment by ryanking** [comment]
Thanks @sjauld.

I should have gone first.

I am Principal Engineer in the Infrastructure and Security group at the [Chan Zuckerberg Initiative](http://chanzuckerberg.com/). My team builds tools and manages infrastructure to support our initiative teams.

Snowflake is our primary data warehouse platform. I started this project to tie together our snowflake permissions management with the rest of our infrastructure (AWS, Okta, etc).

**Comment by grahamplace** [comment]
Hi folks! I'm an engineer currently working on data/analytics tooling at [Opendoor](https://www.opendoor.com/).

We also use Snowflake as our data warehouse platform, and we adopted this provider to manage our Snowflake resources via Terraform like we do other things in our ecosystem. 

We wanted to manage Network Policies via Terraform as well, so I recently made my first contribution to this project: https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/271, which adds the `snowflake_network_policy` and `snowflake_network_policy_attachment` resources 

Hoping to make more contributions in the future!
